distuv/beta.go

```diff
@@ -27,6 +29,17 @@ type Beta struct {
 	Source *rand.Rand
 }
 
+// CDF computes the value of the cumulative density function at x.
+func (b Beta) CDF(x float64) float64 {
+	if x <= 0 {
+		return 0
+	}
+	if x >= 1 {
+		return 1
+	}
+	return mathext.Incbeta(b.Alpha, b.Beta, x)
```

Comment:
Yea, agreed. The documentation suggests it computes the unregularized integral, but these tests hopefully wouldn't pass if it did.